src/typings/strip-json-comments/strip-json-comments.d.ts

```diff
@@ -0,0 +1,21 @@
+// Generated by typings
```

Comment:
I was referring to this typings file. Since you've removed it as a dependency from our package.json, we should remove this typings file to avoid clutter.